src/addon.cpp

```diff
@@ -131,22 +131,24 @@ NAN_METHOD(Addon::configure)
             if (stripTypeValue == "rgb") {
                 ws2811.channel[0].strip_type = WS2811_STRIP_RGB;
             }
-            if (stripTypeValue == "rbg") {
+            else if (stripTypeValue == "rbg") {
                 ws2811.channel[0].strip_type = WS2811_STRIP_RBG;
             }
-            if (stripTypeValue == "grb") {
+            else if (stripTypeValue == "grb") {
                 ws2811.channel[0].strip_type = WS2811_STRIP_GRB;
             }
-            if (stripTypeValue == "gbr") {
+            else if (stripTypeValue == "gbr") {
                 ws2811.channel[0].strip_type = WS2811_STRIP_GBR;
             }
-            if (stripTypeValue == "brg") {
+            else if (stripTypeValue == "brg") {
                 ws2811.channel[0].strip_type = WS2811_STRIP_BRG;
             }
-            if (stripTypeValue == "bgr") {
+            else if (stripTypeValue == "bgr") {
                 ws2811.channel[0].strip_type = WS2811_STRIP_BGR;
             }
-
+            else if (stripTypeValue == "grbw") {
+                ws2811.channel[0].strip_type = SK6812_STRIP_GRBW;
+            }
```

Comment:
The three lines above are the only significant changes.
I have successfully used this code from a NodeJS script to control a strip of SK6812 GRBW LEDs.